taipy/core/_repository/_decoder.py

```diff
@@ -16,7 +16,7 @@
 
 class _Decoder(json.JSONDecoder):
     def __init__(self, *args, **kwargs):
-        json.JSONDecoder.__init__(self, object_hook=self.object_hook, *args, **kwargs)
+        json.JSONDecoder.__init__(self, *args, **kwargs, object_hook=self.object_hook)
```

Comment:
Sure about this one? That looks strange to me

Reply:
The idea is that keyword arguments should be at the end